packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx

```diff
@@ -149,33 +148,20 @@ const OnfidoSdkViewContainer = ({
     const getOnfidoServiceToken = React.useCallback(
         (): Promise<string | { error: TAPIError }> =>
             new Promise(resolve => {
-                const onfido_cookie_name = 'onfido_token';
-                const onfido_cookie = Cookies.get(onfido_cookie_name);
-
-                if (!onfido_cookie) {
-                    WS.serviceToken({
-                        service_token: 1,
-                        service: 'onfido',
-                        country: token_country_code,
-                    }).then((response: TServiceToken) => {
-                        if (response.error) {
-                            resolve({ error: response.error });
-                            return;
-                        }
-                        if (response.service_token?.onfido) {
-                            const { token } = response.service_token.onfido;
-                            const in_90_minutes = 1 / 16;
-                            Cookies.set(onfido_cookie_name, token, {
-                                expires: in_90_minutes,
-                                secure: true,
-                                sameSite: 'strict',
-                            });
-                            resolve(token);
-                        }
-                    });
-                } else {
-                    resolve(onfido_cookie);
-                }
+                WS.serviceToken({
+                    service_token: 1,
+                    service: 'onfido',
+                    country: token_country_code,
+                }).then((response: TServiceToken) => {
```

Comment:
@shahzaib-deriv Out of this PR scope maybe, But I would suggest replacing this with `useFetch` and improving the code a little.

A couple of things can be removed then, The loading and error stuff, The exponential backoff strategy thingy, etc...

cc: @matin-deriv